test/NCalc.Tests/AsyncTests.cs

```diff
@@ -33,7 +33,7 @@ public async Task ShouldEvaluateAsyncFunction()
     [Fact]
     public async Task ShouldEvaluateAsyncParameter()
     {
-        var expression = new AsyncExpression("(a + b) == 'Leo'");
+        var expression = new AsyncExpression("(a + b) == 'Leo'", ExpressionOptions.StringConcat);
```

Comment:
This is the breaking change right? Expressions like this would need ExpressionOptions.StringConcat?